CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 14.2.2 (12/07/23)
+
+* Prevent panic when dialing a deleted Application Server. [#35525](https://github.com/gravitational/teleport/pull/35525)
+* Fixed GCP VM auto-discovery not using instances' internal IP address. [#35521](https://github.com/gravitational/teleport/pull/35521)
+* Calculate latency of Web SSH sessions and report it to users. [#35516](https://github.com/gravitational/teleport/pull/35516)
+* Fix bot's unable to view or approve access requests due to ULS overwriting. [#35512](https://github.com/gravitational/teleport/pull/35512)
```

Comment:
Bots, not bots. 

And users aren't going to know what ULS is.

Reply:
> And users aren't going to know what ULS is.

Neither do I tbh 😄

Reply:
Ah it's "user login state" apparently lol. Yeah I'll remove that.